build/push-ci-build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pushes a continuous integration build to our official CI repository
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+LATEST=$(git describe)
```

Comment:
We talked about this offline and pushed the problem on the client. (Sorry, future clients.) Another way I considered approaching this, BTW, is to actually do something like a "git log --pretty=oneline" and dumping that alongside, so the client can have a sorted record, but I'm not sure what the equivalent is for sorted describe record.